### ISSUE TITLE
Galaxy Store?

### ISSUE THREAD

**Issue by Suncatcher** [beemdevelopment/Aegis#1102]
Any plans to release the app in Galaxy Store?

**Comment by alexbakker** [comment]
No. Why would you like to see Aegis in the Galaxy Store?

**Comment by Suncatcher** [comment]
Maybe because I use Galaxy 😐

**Comment by alexbakker** [comment]
Right, but is there are reason you can't get Aegis from the Google Playstore instead?

I'm just trying the gauge whether publishing to the Galaxy Store is worth the effort. If all devices that have the Galaxy Store also have the Google Playstore, almost nobody is going to install Aegis through the Galaxy Store, would be my guess.

**Comment by Suncatcher** [comment]
> Right, but is there are reason you can't get Aegis from the Google Playstore instead?

yes, it is. I have a Chinese version of the phone where they do not add Google services, only Galaxy framework exists. Not to mention there is big community of people that do not want to install Gapps through ideological reasons
https://www.reddit.com/r/degoogle/
https://nomoregoogle.com/
https://github.com/ungoogled-software/ungoogled-chromium

Why don't you think about such people and totally exclude them from your worldview?

**Comment by michaelschattgen** [comment]
That's why @alexbakker was asking it, please don't make assumptions. We are aware of these communities which is also why we've already made Aegis availabe on F-Droid. Users without Gapps are able to install Aegis through this Google Play Store alternative. 

(Besides that I can't imagine these communities encourage the usage of the Galaxy Store for the same ideological reasons not to use the Google Play Store but that's another discussion)

Personally I don't see a reason to add Aegis to the Galaxy Store but we might consider it when it's more requested.
